Tiltfile

```diff
@@ -224,23 +224,23 @@ def flavors():
         print("{} was not specified in tilt_config.json, attempting to load {}".format(ssh_pub_key, ssh_pub_key_path))
         os.environ.update({ssh_pub_key: base64_encode_file(ssh_pub_key_path)})
 
-    for flavor in cfg.get("worker-flavors", []):
-        if flavor not in worker_templates:
-            worker_templates.append(flavor)
-    for flavor in worker_templates:
-        deploy_worker_templates(flavor, substitutions)
+    templatelist = [ item for item in listdir("./templates") ]
+    for template in templatelist:
+        if template not in worker_templates and os.path.basename(template).startswith("cluster-template"):
```

Comment:
@nader-ziada `and os.path.basename(template).startswith("cluster-template")` should fix the issue you ran into when another file is present

Reply:
working now, even with the .DS_Store file there :)